### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by unixfox** [dan-v/awslambdaproxy#10]
Hello I made a Docker image for awslambdaproxy with an OpenSSH server built in: https://github.com/unixfox/awslambdaproxy-docker

Would you like to integrate it into the official awslambdaproxy repository?

**Comment by dan-v** [comment]
Thanks @unixfox! Sorry for the very long delay in response. I pulled in most of your changes, but made some changes here and there like having it build from source. Open to any additional changes to improve or simplify this further. I think I'd rather it just directly pass all arguments to the awslambdaproxy cli so that separate env variables are not required and it would be easier to use directly from docker cli. Going to leave this open for now until that is further fleshed out.

**Comment by unixfox** [comment]
No worries about the delay, it's fine!

I looked into your changes and it's good for me expect that you removed the ability to pass [Docker secrets](https://docs.docker.com/engine/swarm/secrets/): https://github.com/unixfox/awslambdaproxy-docker/blob/master/ENTRYPOINT.sh#L7

About the environments variables, you are right that it's better to give the ability to pass args instead of having to update the image with new env variables each time new args come up. If I remember well I actually did with the env variables way because I wanted to pass defaults values for things like regions, listeners but it's totally possible to include an `if` in the script that launch the image with default values if no args are provided.
